test/job.js

```diff
@@ -46,15 +52,85 @@ describe('/POST Post a new job', () => {
           response.body.should.have
             .property('message')
             .eql('Create the job successful!');
-          response.body.job.should.have.property('id');
-          response.body.job.should.have
-            .property('title')
-            .eql(jobParameters.title);
-          response.body.job.should.have
-            .property('tags')
-            .eql(jobParameters.tags);
+          id = response.body.job.id;
+          done();
+        });
+    });
+  });
+
+  describe('/PUT Update the job', () => {
```

Comment:
What happens if we don't have "/POST Post a new job" when testing "/PUT Update the job"? Does this test still work?

As I mentioned before, the unit test should be isolated. It shouldn't depend on other unit tests. Otherwise, your tests are unstable and easy to fail.

It's very good that you clean your data after the test run. It should be better if we can set up or prepare the data "before" the test run.